### PR TITLE
Prepare release 2.0.0

### PR DESCRIPTION
**Pull request by abikouo**
##### SUMMARY
- Run `antsibull-changelog` to generate release notes.
- Update `galaxy.yml` version to `2.0.0`

##### ISSUE TYPE
- Release Pull Request

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,39 @@ The cloud.terraform collection Release Notes
 .. contents:: Topics
 
 
+v2.0.0
+======
+
+Major Changes
+-------------
+
+- Added search_child_modules option to terraform_provider inventory plugin.
```
Comment (alinabuzachis) [inline CHANGELOG.rst:14]:
```suggestion
- terraform_provider - Added ``search_child_modules`` option.
```

Comment (alinabuzachis) [reply CHANGELOG.rst:14]: Can you also please link the PR where this feature has been added?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,39 @@ The cloud.terraform collection Release Notes
...
+- Allow project_path in terraform_provider inventory plugin to accept a list of paths.
```
Comment (alinabuzachis) [inline CHANGELOG.rst:15]: Same here for the PR.
```suggestion
- terraform_provider  - Allow ``project_path`` in terraform_provider inventory plugin to accept a list of paths.
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -5,6 +5,39 @@ The cloud.terraform collection Release Notes
...
+
+Minor Changes
+-------------
+
+- Removed integration tests workaround in terrform_provider. (https://github.com/ansible-collections/cloud.terraform/pull/84)
+- Set default of state_file in terraform_provider inventory plugin to a blank string.
```
Comment (alinabuzachis) [inline CHANGELOG.rst:21]: Is this a breaking change? Can you also please link the PR?
```suggestion
- terraform_provider  - Set default of ``state_file`` to a blank string.
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -5,6 +5,39 @@ The cloud.terraform collection Release Notes
...
+- add support for workspace when running `terraform output` in modules `cloud.terraform.terraform`, `cloud.terraform.terraform_output` and lookup `cloud.terraform.tf_output` (https://github.com/ansible-collections/cloud.terraform/issues/85).
```
Comment (alinabuzachis) [inline CHANGELOG.rst:22]: We should also update version_added for this option https://github.com/ansible-collections/cloud.terraform/pull/88/files#diff-fedb790d9a5867b094724ba9d63a33283da12b08bbfe9123f121d805a14c541eR39.
```suggestion
- terraform  - add support for ``workspace`` when running ``terraform output`` (https://github.com/ansible-collections/cloud.terraform/issues/85). 
- terraform_output -  add support for ``workspace`` when running ``terraform output`` (https://github.com/ansible-collections/cloud.terraform/issues/85).
- tf_output - add support for ``workspace`` when running ``terraform output`` (https://github.com/ansible-collections/cloud.terraform/issues/85).
```

File: CHANGELOG.rst [hunk 5]
```diff
@@ -5,6 +5,39 @@ The cloud.terraform collection Release Notes
...
+
+Breaking Changes / Porting Guide
+--------------------------------
+
+- Remove support for ansible-core < 2.14 (https://github.com/ansible-collections/cloud.terraform/pull/102).
+
+Bugfixes
+--------
+
+- Accept Terraform executables present on PATH passed in as binary_path without specifying their absolute path. (https://github.com/ansible-collections/cloud.terraform/issues/49)
```
Comment (alinabuzachis) [inline CHANGELOG.rst:33]:
```suggestion
- module_utils - Accept Terraform executables present on PATH passed in as ``binary_path`` without specifying their absolute path. (https://github.com/ansible-collections/cloud.terraform/issues/49)
```

File: CHANGELOG.rst [hunk 6]
```diff
@@ -5,6 +5,39 @@ The cloud.terraform collection Release Notes
...
+- Allow `nested_type` attributes in terraform schema. (https://github.com/ansible-collections/cloud.terraform/issues/93)
```
Comment (alinabuzachis) [inline CHANGELOG.rst:34]:
```suggestion
- module_utils - Allow ``nested_type`` attribute in terraform schema. (https://github.com/ansible-collections/cloud.terraform/issues/93)
```

File: CHANGELOG.rst [hunk 7]
```diff
@@ -5,6 +5,39 @@ The cloud.terraform collection Release Notes
...
+- Fix AWS SQS queue creation. The `values` attribute in terraform output is optional. SQS is one of (rare) cases where `values` is absent. (https://github.com/ansible-collections/cloud.terraform/issues/86)
```
Comment (alinabuzachis) [inline CHANGELOG.rst:35]:
```suggestion
- module_utils - Fix AWS SQS queue creation. The ``values`` attribute in terraform output is optional. SQS is one of (rare) cases where ``values`` is absent. (https://github.com/ansible-collections/cloud.terraform/issues/86)
```

File: CHANGELOG.rst [hunk 8]
```diff
@@ -5,6 +5,39 @@ The cloud.terraform collection Release Notes
...
+- fix integration test target `terraform_provider`, add step to copy example modules into terraform working directory (https://github.com/ansible-collections/cloud.terraform/pull/89).
```
Comment (alinabuzachis) [inline CHANGELOG.rst:36]: I guess this one should be probably documented as trivial.

File: CHANGELOG.rst [hunk 9]
```diff
@@ -5,6 +5,39 @@ The cloud.terraform collection Release Notes
...
+- fix spaces between characters in command field in result (https://github.com/ansible-collections/cloud.terraform/pull/76).
```
Comment (alinabuzachis) [inline CHANGELOG.rst:37]:
```suggestion
- terraform -  fix spaces between characters in command field in result (https://github.com/ansible-collections/cloud.terraform/pull/76).
```

**Comment by alinabuzachis** [comment]
I am also wondering if we should also regenerate the docs for the plugins since it's a new release.

File: CHANGELOG.rst (hunk 1 above)
Comment (abikouo) [reply CHANGELOG.rst:14]: I will do it, but we are supposed to track this during PR review

**Comment by abikouo** [comment]
> I am also wondering if we should also regenerate the docs for the plugins since it's a new release.

I have updated the plugin documentation. Thanks for the reviews @alinabuzachis

File: changelogs/changelog.yaml [hunk 10]
```diff
@@ -70,3 +70,48 @@ releases:
         name: terraform_provider
         namespace: null
     release_date: '2023-03-23'
+  2.0.0:
+    changes:
+      breaking_changes:
+      - Remove support for ansible-core < 2.14 (https://github.com/ansible-collections/cloud.terraform/pull/102).
+      bugfixes:
+      - Accept Terraform executables present on PATH passed in as binary_path without
```
Comment (alinabuzachis) [inline changelogs/changelog.yaml:78]: You also need to update here.

Comment (abikouo) [reply changelogs/changelog.yaml]: done

File: CHANGELOG.rst [hunk 11]
```diff
@@ -5,6 +5,40 @@ The cloud.terraform collection Release Notes
...
+-------------
+
+- Removed integration tests workaround in terrform_provider. (https://github.com/ansible-collections/cloud.terraform/pull/84)
+- Set default of ``state_file`` in terraform_provider inventory plugin to a blank string (https://github.com/ansible-collections/cloud.terraform/pull/55).
+- terraform  - add support for ``workspace`` when running ``terraform output`` (https://github.com/ansible-collections/cloud.terraform/issues/85). 
+- terraform_output -  add support for ``workspace`` when running ``terraform output`` (https://github.com/ansible-collections/cloud.terraform/issues/85).
+- tf_output - add support for ``workspace`` when running ``terraform output`` (https://github.com/ansible-collections/cloud.terraform/issues/85).
+- terraform - cleanup temporary file create during module execution. (https://github.com/ansible-collections/cloud.terraform/issues/2)
+
+Breaking Changes / Porting Guide
```
Comment (hakbailey) [inline CHANGELOG.rst:27]: Should we move breaking changes to the top (in the config.yaml file)?

**Review by hakbailey — APPROVED** [review]

File: CHANGELOG.rst (hunk 11 above)
Comment (abikouo) [reply CHANGELOG.rst]: done

**Review by hakbailey — APPROVED** [review]

**Review by jillr — APPROVED** [review]

**Pull request merged**